src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -1,45 +1,57 @@
 <kbn-management-app section="kibana">
   <kbn-management-indices>
-    <div ng-controller="managementIndicesCreate" class="kbn-management-indices-create">
```

Comment:
That's actually not a class in `main.less`, it's an element selector. Leaving it for now.